src/content/core/okteto-variables.mdx

```diff
@@ -203,6 +203,7 @@ Okteto provides a list of variables that serve as feature flags for some of the
 - `OKTETO_FORCE_REMOTE`:
 - `OKTETO_FOLDER`:
 - `OKTETO_REMOTE_CLI_IMAGE`:
+- `OKTETO_LOCAL_REGISTRY_STORE_ENABLED`: enables the local registry credentials and uses them instead of the one stored in the cluster.
```

Comment:
```suggestion
- `OKTETO_LOCAL_REGISTRY_STORE_ENABLED`: enables the registry credentials stored locally and uses them instead of the one stored in the cluster.
```

In the other form I was understanding at the beginning that the registry was local. WDYT @codyjlandstrom ?

Reply:
It's not one or the other, it's to use the local credentials over cluster credentials, but if you don't have a local entry for the cluster credentials, the cluster credentials are used. We should also document the default value